query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/PostgresBasedRequestHandler.java

```diff
@@ -520,28 +520,27 @@ private void handleSelection(
     } while (resultSet.next());
   }
 
+  // TODO - Need to validate and fix this function
```

Comment:
Yes, we will need only single parse function in Postgres impl

Reply:
As part of follow up